### PR TITLE
Explain difference between Elixir x.y and x.y.z

### PR DESCRIPTION
**Pull request by BanzaiMan**
See https://github.com/elixir-lang/elixir/issues/6618 for discussion.

### CODE REVIEW

File: user/languages/elixir.md [hunk 1]
```diff
@@ -44,6 +44,30 @@ language: elixir
 ```
 {: data-file=".travis.yml"}
 
+### Specify which Elixir version to test with
```
Comment (plaindocs) [inline user/languages/elixir.md]: We should probably default to to 'build' instead of 'test' here?

**Comment by plaindocs** [comment]
Looks good to me. 

Are we quoting version numbers in the yaml specifically here for a reason?

File: user/languages/elixir.md (hunk 1 above)
Comment (BanzaiMan) [reply user/languages/elixir.md]: Agreed.

**Comment by BanzaiMan** [comment]
Quotation is for consistency; `1.5` is technically a float, not a string. I know this is not consistently enforced in our docs, but we should try to be more consistent nonetheless.

**Comment by plaindocs** [comment]
So we should be quoting *all* version numbers as strings in yaml? Roger that.

**Pull request merged**